dags/run_all_processors.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+from airflow.models import DAG
+from airflow.operators.latest_only_operator import LatestOnlyOperator
+import utils.helpers as helpers
+
+args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime.datetime(2017, 1, 1),
```

Comment:
I wonder what happens if we don't mention a start date. According to the docs, it's optional, and since it's a "on demand" DAG, it shouldn't have any dates attached to it. Same would apply to the other on-demand DAGs we have.